starbucks_dw/models/reporting/report_channel_effectiveness.sql

```diff
@@ -0,0 +1,42 @@
+{{
+  config(
+    materialized = 'table',
+    )
+}}
+
+WITH prep_offer_response AS (
+
+SELECT
+offer_channel,
+offer_type,
+{{ dbt_utils.pivot(
+      'transaction_status',
+      dbt_utils.get_column_values(ref('fct_offer_transactions'), 'transaction_status')
```

Comment:
Your build failed due an issue on this line. You missed the way to pass the arguments to `get_column_values`.

Documentation: https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#get_column_values-source

```suggestion
      dbt_utils.get_column_values(table=ref('fct_offer_transactions'), column='transaction_status')
```